src/WordPress/AsyncHttp/async_http_streams.php

```diff
@@ -230,7 +230,7 @@ function stream_http_prepare_request_bytes( $url ) {
 
 	// @TODO: Add support for Accept-Encoding: gzip
 
-	return str_replace( "\n", "\r\n", $request ) . "\r\n\r\n";
+	return $request . "\r\n\r\n";
```

Comment:
It seems like we'd still get inconsistent newline style depending on where this file was last saved and where is it used. Let's declare each line of the request separately and concatenate them with the \r\n sequence so the result is always the same.

Reply:
The newest change introduces single newline style \r\n.